docs/environment/custom-domains.md

```diff
@@ -31,9 +31,10 @@ After validating the domain and the certificate we can now link the custom domai
 - click "Create"
 - enter your domain name, select the certificate you created above and save
 - edit the domain that was created
-- add a "Path mapping" to map the `/` URL (or any URL you want) to your HTTP application in the `Prod` stage, for example:
+- click "Configure API mappings" to add an "API mapping" which maps the `/` URL (or any URL you want) to your HTTP application in the `Dev` stage, for example:
 
   ![](custom-domains-path-mapping.png)
+If you are using Cloudfront:
 - after clicking "Save", note the URL in the `Target Domain Name` section (`<random>.cloudfront.net`)
```

Comment:
```suggestion
- after saving the "API mappings", find the `API Gateway domain name` in the "Configurations" tab (`<random>.cloudfront.net`)
```

This is what I mean:

![CleanShot 2020-11-03 at 11 50 59](https://user-images.githubusercontent.com/720328/97976392-eeaf5780-1dca-11eb-8ee8-1071b42f19d0.png)

Reply:
Mine looks a bit different, though:

![Screenshot 2020-11-03 at 13 54 34](https://user-images.githubusercontent.com/787653/97982573-b153c580-1ddc-11eb-85e6-ee2edd9cd833.png)

Reply:
Yes, ok here is an alternative suggestion:

```suggestion
- after saving the "API mappings", find the `API Gateway domain name` in the "Configurations" tab
```